Makefile

```diff
@@ -5,6 +5,11 @@ ifndef MICROOVN_SNAP_CHANNEL
 	export MICROOVN_SNAP_CHANNEL="22.03/stable"
 endif
 
+ALL_TESTS := $(wildcard tests/*.bats)
+MICROOVN_SOURCES := $(shell find microovn/ -type f)
```

Comment:
Neat trick, and I guess this is OK since we're only using it for detecting whether a (re-)build is required.  We would need to explicitly list files if we want to use it for something like `make dist` or similar in the future, but I see no need for that.